DigitalOffice/Shared/Kernel/Broker/OperationResultWrapper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace LT.DigitalOffice.Kernel.Broker
+{
+    /// <summary>
+    /// Wraps the returned object from a function in OperationResult.
+    /// </summary>
+    /// <typeparam name="T">parameter of the called function</typeparam>
+    /// <typeparam name="TRes">response from the called function, which will be wrapped in OperationResult</typeparam>
+    public class OperationResultWrapper
+    {
+        public static object CreateResponse<T, TRes>(Func<T, TRes> func, T arg)
+        {
+            try
+            {
+                return new
+                {
+                    Body = func.Invoke(arg),
+                    IsSuccess = true
+                };
+            }
+            //TODO add handler for others exceptions and custom exceptions
+            catch (Exception e)
+            {
+                return new
+                {
+                    IsSuccess = false
+                };
+            }
+        }
+    }
```

Comment:
I think this class is overkill (extra, unnecessary).
We only use MassTransit Response/Request logic in one place: when we send or consume the message.
There is no need for extra classes, methods. 
I've watched Spartak's video tutorial and it's crystal clear there. No extra code, no extra methods etc. So I disagree on this.

Reply:
How to catch errors that are thrown and then send them through mass transit?

Reply:
You can use try, catch in Consumer, for example.

Reply:
to not duplicate logic in several services, I moved it to this class